packages/core-mobile/app/services/earn/EarnService.ts

```diff
@@ -206,7 +222,10 @@ class EarnService {
       avaxPNetwork.networkToken.decimals,
       avaxPNetwork.networkToken.symbol
     )
-    const unmintedSupply = supplyCap.sub(currentSupply)
+    let unmintedSupply = supplyCap.sub(currentSupply)
+    if (unmintedSupply.lt(0)) {
+      unmintedSupply = unmintedSupply.mul(-1)
+    }
```

Comment:
not sure if this is just a glitch in p devnet, the supply cap is somehow less than the current supply.  which shouldn't happen.

Reply:
we should probably throw here

Reply:
https://ava-labs.atlassian.net/browse/CP-9539